packages/@okta/vuepress-site/docs/release-notes/2020-09-4/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Okta API Products Release Notes
+---
+
+## 2020.09.4
+
+| Change                                                    | Expected in Preview Orgs |
+| --------------------------------------------------------- | ------------------------ |
+| [Bugs fixed in 2020.09.4](#bug-fixed-in-2020-09-4)       | September 30, 2020           |
+
+### Bugs fixed in 2020.09.4
+
+* When an OAuth service client called the [`/v1/authorize` endpoint](/docs/reference/api/oidc/#authorize), the returned error description was inaccurate. (OKTA-252750)
```

Comment:
Usually, we leave out the `/v1` when giving the name of an endpoint, so it would be `/authorize` here--unless there was a reason to specify v1?

Reply:
For formatting, I think we usually would have just the word "endpoint" as the link text. the name of the endpoint that comes before that would not be part of the link text (i.e., it wouldn't be in the square brackets).

Reply:
For your first comment, this is an OIDC endpoint, so it's a bit different. The doc has the URL defined as:
${baseUrl}/v1/authorize, but it calls it /authorize. Based on that, I think your call of just `/authorize` is the right way to go.